container/worker/run_openqa_worker.sh

```diff
@@ -9,7 +9,9 @@ fi
 /root/qemu/kvm-mknod.sh
 
 if [ -e "/dev/kvm" ] && getent group kvm > /dev/null; then
-  groupmod -g "$(ls -lhn /dev/kvm | cut -d ' ' -f 4)" kvm
+  group=$(ls -lhn /dev/kvm | cut -d ' ' -f 4)
+  [[ $(getent group kvm | cut -d ":" -f3) -ne "$group" ]] && groupmod -g "$group" --non-unique kvm
```

Comment:
The groupmod command will not do any harm if the group ID won't change so I'd get rid of the check:
```suggestion
groupmod -g "$group" --non-unique kvm
```